src/components/Toast/examples.tsx

```
@@ -0,0 +1,449 @@
+      // Here you would make the API call to delete permanently
+      console.log('Permanently deleted:', itemName);
+    }, 7000);
+  };
+
+  return (
+    <div>
+      <h2>Items</h2>
+      {items.map((item) => (
+        <div key={item}>
+          {item}
+          <button onClick={() => handleDelete(item)}>Delete</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)